crates/next-core/src/next_font_google/util.rs

```diff
@@ -365,4 +419,25 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_variable_font_without_wgth_axis() -> Result<()> {
+        assert_eq!(
+            get_stylesheet_url(
+                "Roboto Serif",
+                &FontAxes {
+                    wght: indexset! {},
+                    ital: indexset! {},
+                    variable_axes: Some(vec![
+                        ("EDPT".to_owned(), "0..200".to_owned()),
+                        ("EHLT".to_owned(), "0..24".to_owned()),
+                    ])
+                },
+                "optional"
+            )?,
+            "https://fonts.googleapis.com/css2?family=Roboto+Serif&display=optional"
```

Comment:
Oh sorry, I messed up my example test. The provided axes should still be included in the url.
```
https://fonts.googleapis.com/css2?family=Nabla:EDPT,EHLT@0..200,0..24&display=optional
```